### ISSUE TITLE
Potential Issue with Aspect Ratio Distortion on Hand Dataset

### ISSUE THREAD

**Issue by Djanghao** [MIRACLE-Center/Oneshot_landmark_detection#13]
I have a question regarding the compute_spacing method used on the HAND dataset. Here is the code:

https://github.com/MIRACLE-Center/Oneshot_landmark_detection/blob/6c94958c119c461d8110e31d704f261e9cfef87e/code/cc2d-final/datasets/hand.py#L79-L82

In this method, the same scaling factor is applied to both the x-axis and y-axis, but I noticed that the images in the HAND dataset are not square. During the preprocessing, the images are resized to 384x384, which suggests that the different scale factors are used for both axes.

Could this resizing affect the accuracy of Mean Radial Error (MRE) calculations?

Thank you for your time and assistance.

**Comment by qsyao** [comment]
Yes, it is indeed quite a problem, we just follow our baseline for computing spacing, which assumes that the spacing is the same for x and y axis ... We find that the X and Y spacing of most of the hand radiographs are the same.
